lib/declarations.d.ts

```diff
@@ -409,8 +411,17 @@ interface IPluginsService {
 	printPlugins(plugins: IPlugin[]): void;
 	addPlugin(pluginName: string): IFuture<void>;
 	removePlugin(pluginName: string): IFuture<void>;
-	configurePlugin(pluginName: string, configuration?: string): IFuture<void>;
+	/**
+	 * Used to configure a plugin.
+	 * @param  {string} pluginName The name of the plugin.
+	 * @param  {string} version The version of the plugin.
+	 * @param  {string[]} configurations Configurations in which the plugin should be configured. Example: ['debug'], ['debug', 'release']
+	 * @return {IFuture<void>}
+	 */
+	configurePlugin(pluginName: string, version?: string, configurations?: string[]): IFuture<void>;
 	isPluginInstalled(pluginName: string): boolean;
+	getPluginBasicInformation(pluginName: string): IBasicPluginInformation;
```

Comment:
it will be great to add documentation for all methods, that you've added to this interface :smiley_cat: